apps/web/prisma/schema.prisma

```diff
@@ -418,10 +422,46 @@ model ExecutedAction {
   url     String?
 
   // additional fields as a result of the action
-  draftId      String? // Gmail draft ID created by DRAFT_EMAIL action
-  wasDraftSent Boolean? // Tracks if the corresponding draft was sent (true) or ignored/superseded (false)
-  draftSendLog DraftSendLog? // Will exist if the draft was sent
-  digestItems  DigestItem[] // Relation to digest items created by this action
+  draftId         String? // Gmail draft ID created by DRAFT_EMAIL action
+  wasDraftSent    Boolean? // Tracks if the corresponding draft was sent (true) or ignored/superseded (false)
+  draftSendLog    DraftSendLog? // Will exist if the draft was sent
+  digestItems     DigestItem[] // Relation to digest items created by this action
+  scheduledAction ScheduledAction? // Reverse relation for delayed actions
+}
+
+model ScheduledAction {
+  id             String                @id @default(cuid())
+  executedRuleId String
+  actionType     ActionType
+  messageId      String
+  threadId       String
+  scheduledFor   DateTime
+  emailAccountId String
+  status         ScheduledActionStatus @default(PENDING)
+
+  label   String?
+  subject String?
+  content String?
+  to      String?
+  cc      String?
+  bcc     String?
+  url     String?
+
+  qstashMessageId String? // QStash message ID for efficient cancellation
+
+  executedAt       DateTime?
+  executedActionId String?   @unique
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
```

Comment:
these are always the first fields in all other models. let's be consistent and move it up